resolver/src/test/java/org/apache/james/jspf/SynchronousSPFExecutorIntegrationTest.java

```diff
@@ -67,10 +79,27 @@ public void shouldHandleDomainNotFound() {
     @Test
     public void shouldHandleSPFNotFound() {
         SPF spf = DefaultSPF.createSync();
-        SPFResult result = spf.checkSPF("207.54.72.202","do_not_reply@de","de");
+        SPFResult result = spf.checkSPF("207.54.72.202","do_not_reply@com.br","com.br");
         System.out.println(result.getResult());
         System.out.println(result.getExplanation());
         System.out.println(result.getHeader());
         assertEquals("none", result.getResult());
     }
+
+    @Test
+    public void shouldReturnTempErrorOnPortUnreachable() throws UnknownHostException {
+        Resolver simpleResolver = new SimpleResolver("127.0.0.1");
+        simpleResolver.setPort(ThreadLocalRandom.current().nextInt(55000, 56000));
+
+        DNSServiceXBillImpl dns = new DNSServiceXBillImpl(simpleResolver);
+
+        SPF spf = new SPF(dns, new SynchronousSPFExecutor(dns));
+        SPFResult result = spf.checkSPF("207.54.72.202",
+                "do_not_reply@reyifglerifwukfvbdjhrkbvebvekvfulervkerkeruerbeb.de",
+                "reyifglerifwukfvbdjhrkbvebvekvfulervkerkeruerbeb.de");
+        System.out.println(result.getResult());
+        System.out.println(result.getExplanation());
+        System.out.println(result.getHeader());
```

Comment:
Idem

Reply:
In some tests like these, the details helps sometimes. Do you prefer I replace all these println with logger instead of remove them ?

Reply:
We could replace those with assertions so that the test reader get it in the source code of the test?

Reply:
Right. I will replace with assert the most relevant, and remove others.